R/do_groupChromPeaks-functions.R

```diff
@@ -240,7 +241,7 @@ do_groupChromPeaks_density_par <- function(peaks, sampleGroups,
                                    sampleGroupTbl, minFr, minSmpls,
                                    sampleGroupNms, peakOrdr) {
         den <- density(z[, "rt"], bw = bw, from = rtr[1] - 3 * bw,
-                       to = rtr[2] + 3 * bw)
+                       to = rtr[2] + 3 * bw, n = max(512, 2^(ceiling(log2(diff(rtr)/(bw/2))))))
```

Comment:
Ok, n=512 is the default, and one might want to increase the number of points for large RT ranges and/or small bw ?